### PR TITLE
[FIX] website_slides: fix multiple course review bug

### PR DESCRIPTION
**Pull request by pko-odoo**
Before this commit, it was possible for one to post multiple reviews when all the already posted reviews had the 'Employee Only' state. A traceback was also thrown when trying to update an 'Employee Only'
comment. It was also possible to edit a log note.
This commit fixes all these issues.

Task-2810085

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website_slides/static/src/tests/tours/slides_course_reviews.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+odoo.define('website_slides.tour.slide.course.reviews', function (require) {
+'use strict';
+
+var tour = require('web_tour.tour');
```
Comment (std-odoo) [inline addons/website_slides/static/src/tests/tours/slides_course_reviews.js:4]:
```suggestion
const tour = require('web_tour.tour');

File: addons/website_slides/tests/test_ui_wslides.py [hunk 2]
```diff
@@ -148,6 +148,25 @@ def test_full_screen_edition_website_publisher(self):
             'odoo.__DEBUG__.services["web_tour.tour"].tours.full_screen_web_editor.ready',
             login=user_demo.login)
 
+    def test_course_reviews_elearning_officer(self):
+        user_demo = self.user_demo
+        user_demo.flush()
```
Comment (std-odoo) [inline addons/website_slides/tests/test_ui_wslides.py:153]: why do you need to flush ?

**Review by std-odoo — COMMENTED** [review]
@pko-odoo Hi :) 

2 small comments, otherwise LGTM :)

File: addons/website_slides/tests/test_ui_wslides.py (hunk 2 above)
Comment (pko-odoo) [reply addons/website_slides/tests/test_ui_wslides.py]: Actually I copy/pasted the code above and didn't think about it ^^
I removed it

**Comment by std-odoo** [comment]
@fwbot up to 14.0

**Comment by std-odoo** [comment]
@fwbot upto 14.0

**Comment by std-odoo** [comment]
@fw-bot ignore

**Comment by std-odoo** [comment]
@robodoo r+

**Pull request closed without merging**